app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -9,13 +9,13 @@
 
 - when 'UsersRole'
   - users_role = current_or_last_object_state(version.item_type, version.item_id)
-  - role = Role.find_by(id: users_role.role_id) if users_role
+  - role = Role.find_by(id: users_role.role_id) || PaperTrail::Version.where(item_type: 'Role', item_id: version.item_id).where(id: users_role.role_id)
```

Comment:
While you are at it, you may want to remove the (Role Deleted) and (Organization Deleted) and substitute them with something more meaningful, like we do in the rest of the cases.